### ISSUE TITLE
CampaignTrackingService NullPointerException, v4.4.52

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gzog/analytics-issues#440]
```
Android Google Play Service v4.4.52

This crash error does not happen always.

0 java.lang.NullPointerException
1 at 
com.google.android.gms.analytics.CampaignTrackingService.processIntent(Unknown 
Source)
2 at 
com.google.android.gms.analytics.CampaignTrackingService.onHandleIntent(Unknown 
Source)
3 at 
android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:65)
4 at android.os.Handler.dispatchMessage(Handler.java:99)
5 at android.os.Looper.loop(Looper.java:137)
6 at android.os.HandlerThread.run(HandlerThread.java:60)
```

Original issue reported on code.google.com by `a0s...@gmail.com` on 10 Jun 2014 at 12:21

**Comment by GoogleCodeExporter** [comment]
```
Came across the exact same issue with Nexus 5 running 4.4.2 and Galaxy Nexus 
running 4.1.1
```

Original comment by `boss.a...@gmail.com` on 4 Jul 2014 at 6:51

**Comment by GoogleCodeExporter** [comment]
```
Even i got this issue today.
```

Original comment by `venkat20...@gmail.com` on 15 Jul 2014 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
Hello there!

Can someone check this on rooted device?
See this comment: 
https://productforums.google.com/d/msg/analytics/po1006amU1c/QeEPXuZxN2sJ
```

Original comment by `da...@geak.ru` on 28 Aug 2014 at 4:16

**Comment by GoogleCodeExporter** [comment]
Original comment by `pfrise...@google.com` on 24 Oct 2014 at 7:06
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Release Version 4.0.6 - Google Play Services 7.0 (March 19, 2015)
see 
https://developers.google.com/analytics/devguides/collection/android/changelog
```

Original comment by `pfrise...@google.com` on 23 Mar 2015 at 4:06
- Changed state: **Fixed**
